### DISCUSSION TITLE
Request for  Wheeled Robot RL Example

### DISCUSSION THREAD

**Discussion by mshoaibb** [isaac-sim/IsaacLab#505] [Ideas]
Hi,

I'm working on a Wheeled robot (2-wheeled), but due to the lack of any example in Isaac Lab or Orbit. I'm facing many problems.

Please also introduce a Wheeled robot as an example for RL.

I appreciate any help you can provide.

**Comment by KUsaka-xu** [comment]
https://github.com/jaykorea/Isaac-RL-Two-wheel-Legged-Bot this project may help you
